### DISCUSSION TITLE
For each run, a TIC vs time plot

### DISCUSSION THREAD

**Discussion by KeepCalmAndRushB** [MannLabs/alphadia#155] [Ideas]
Can you please generate a small PNG or JPG from each run, for a fast check?

Something like:

![example](https://github.com/MannLabs/alphadia/assets/43036737/ddb03ad4-d21d-4135-b4dd-996d0c0410c6)

**Comment by mschwoer** [comment]
we see this as a feature in AlphaRaw rather than AlphaDIA ..
tracked internally here https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=-last-updated%3A14days&pane=issue&itemId=111357188
